app/models/concerns/filter/control_sections.rb

```diff
@@ -31,7 +31,18 @@ def control_sections
           short_label: 'Project Type',
           value: describe_project_type_control_section,
         )
-        add_coordinated_entry_sections(section)
+        section.add_control(
+          id: 'coordinated_assessment_living_situation_homeless',
+          label: 'Including CE homeless at entry?',
+          value: @filter.coordinated_assessment_living_situation_homeless ? 'Yes' : nil,
+          hint: "Including Coordinated Entry enrollments where the prior living situation is homeless (#{HUD.homeless_situations(as: :prior).to_sentence}) will include these clients even if they do not have an enrollment in one of the chosen project types.",
+        )
+        section.add_control(
+          id: 'ce_cls_as_homeless',
+          label: 'Including CE Current Living Situation Homeless',
+          value: @filter.ce_cls_as_homeless ? 'Yes' : nil,
+          hint: "Including Coordinated Entry enrollments where the client has at least two homeless current living situations (#{HUD.homeless_situations(as: :current).to_sentence}) within the report range. These clients will be included even if they do not have an enrollment in one of the chosen project types.",
+        )
```

Comment:
I don't think these controls will work for the homeless summary report and should be removed. Since this is a shared concern, including them either needs to be parameterized, or it needs to forked off for that report?

I'm thinking parameterized would be better since this isn't the first report we've seen that is SPM driven

Reply:
I think you're reviewing an outdated version– I discussed with Elliot yesterday and we decided to remove the CE filters